JPS_BASE_LIB/README.md

```diff
@@ -1,12 +1,5 @@
+# The JPS Base Lib is no more
 
-# JPS Base Library
+The artist formerly known as JPS_BASE_LIB has been absorbed by [The World Avatar](https://github.com/TheWorldAvatar/BaseLib).
```

Comment:
I think "absorbed" is a bit misleading here as the code is becoming less of a part of something rather than more.
```suggestion
The artist formerly known as JPS_BASE_LIB has been moved to its own repository within [The World Avatar](https://github.com/TheWorldAvatar/BaseLib).
```